models/cm_default.h

```diff
@@ -0,0 +1,328 @@
+ g_L        uS      Leak conductance of the compartment (default: 0.1 uS)
+ e_L        mV      Leak reversal of the compartment (default: -70. mV)
+=========== ======= ===============================================================
+
+Ion channels and receptor types for the default model are hardcoded.
+For ion channels, there is a Na-channel and a K-channel. Parameters can be set
+by specifying the following entries in the `SetStatus` dictionary argument:
+
+=========== ======= ===========================================================
+ gbar_Na    uS      Maximal conductance Na channel (default: 0 uS)
+ e_Na       mV      Reversal Na channel default (default: 50 mV)
+ gbar_K     uS      Maximal conductance K channel (default: 0 uS)
+ e_K        mV      Reversal K channel (default: 50 mV)
+=========== ======= ===========================================================
+
+For receptors, the choice is from 'AMPA', 'GABA' or 'NMDA' or 'AMPA_NMDA'.
+Ion channels and receptor types can be customized with NESTML.
+
+If `receptor_type` is AMPA
+
+=========== ======= ===========================================================
+ e_AMPA     mV      AMPA reversal (default 0 mV)
+ tau_r_AMPA ms      AMPA rise time (default .2 ms)
+ tau_d_AMPA ms      AMPA decay time (default 3. ms)
+=========== ======= ===========================================================
+
+If `receptor_type` is GABA
+
+=========== ======= ===========================================================
+ e_GABA     mV      GABA reversal (default 0 mV)
+ tau_r_GABA ms      GABA rise time (default .2 ms)
+ tau_d_GABA ms      GABA decay time (default 10. ms)
+=========== ======= ===========================================================
+
+If `receptor_type` is NMDA
+
+=========== ======= ===========================================================
+ e_NMDA     mV      NMDA reversal (default 0 mV)
+ tau_r_NMDA ms      NMDA rise time (default .2 ms)
+ tau_d_NMDA ms      NMDA decay time (default 43. ms)
+=========== ======= ===========================================================
+
+If `receptor_type` is AMPA_NMDA
+
+============ ======= ===========================================================
+ e_AMPA_NMDA mV      NMDA reversal (default 0 mV)
+ tau_r_AMPA  ms      AMPA rise time (default .2 ms)
+ tau_d_AMPA  ms      AMPA decay time (default 3. ms)
+ tau_r_NMDA  ms      NMDA rise time (default .2 ms)
+ tau_d_NMDA  ms      NMDA decay time (default 43. ms)
+ NMDA_ratio  (1)     Ratio of NMDA versus AMPA channels
+============ ======= ===========================================================
+
+Sends
++++++
+
+SpikeEvent
+
+Receives
+++++++++
+
+SpikeEvent, CurrentEvent, DataLoggingRequest
+
+References
+++++++++++
+
+Data-driven reduction of dendritic morphologies with preserved dendro-somatic responses
+WAM Wybo, J Jordan, B Ellenberger, UM Mengual, T Nevian, W Senn
+Elife 10, `e60936 <https://elifesciences.org/articles/60936>`_
+
+See also
+++++++++
+
+NEURON simulator ;-D
```

Comment:
This section will be removed by the doc processor, but as it's more of an inside joke anyway, I'm fine with leaving it here.